dll/win32/shell32/CDefViewUtil.cpp

```diff
@@ -0,0 +1,132 @@
+/*
+ * PROJECT:     shell32
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     DefView utility functions
+ * COPYRIGHT:   Copyright 2024 Whindmar Saksit <whindsaks@proton.me>
+ */
+
+#include "precomp.h"
+
+WINE_DEFAULT_DEBUG_CHANNEL(shell);
+
+class CObjectWithSiteBase :
+    public IObjectWithSite
+{
+    public:
```

Comment:
You can reduce one indentation level less.

Reply:
Do you mean placing `public:` in column 0?

Reply:
Yes this is what I was suggesting (we do that already in other places).